### DISCUSSION TITLE
Quasar Minimalist Design

### DISCUSSION THREAD

**Discussion by pratik227** [quasarframework/quasar#15820] [Show and Tell]
Hello Everyone,

Quasar Minimalist Design is a premium admin template that adopts a minimalist design philosophy. It offers a user-friendly and efficient dashboard for businesses and organisation's, featuring components such as dashboard, customers, library, settings, calendar, and login page. Check out the website for more details. 

Demo - https://quasar-minimalist-design.netlify.app/

For More Details visit  https://github.com/Quasar-Admin-Templates/Quasar-Minimalist-Design

Thanks,
Pratik Patel

**Comment by FadhiliNjagi** [comment]
Just wow! I'm speechless

**Reply by pratik227** [reply]
Thanks
